File: Level 0/코딩 기초 트레이닝/조건 문자열/solution.js
const operations = {
  ">=": (n, m) => n >= m,
  ">!": (n, m) => n > m,
  "<=": (n, m) => n <= m,
  "<!": (n, m) => n < m,
};

function solution(ineq, eq, n, m) {
  const operation = operations[ineq + eq];
  return Number(operation(n, m));
}

File: Level 0/코딩 기초 트레이닝/조건 문자열/solution.py
operations = {
    ">=": lambda n, m: n >= m,
    ">!": lambda n, m: n > m,
    "<=": lambda n, m: n <= m,
    "<!": lambda n, m: n < m,
}


def solution(ineq, eq, n, m):
    operation = operations[ineq + eq]
    return int(operation(n, m))
